src/components/inputWithAutocomplete.jsx

```diff
@@ -5,56 +5,56 @@ class InputWithAutocomplete extends React.Component {
         super(props);
 
         this.state = {
-            isVisibleHints: false
+            hints: []
```

Comment:
That's actually a great idea to replace flag with just an array. I thought about having both of them, but your solution is better. Love it :heart_eyes: